xyz.audiostellar.AudioStellar.yml

```diff
@@ -0,0 +1,344 @@
+
+  - name: liburiparser
+    buildsystem: cmake-ninja
+    config-opts:
+    - "-DURIPARSER_BUILD_DOCS=OFF"
+    - "-DURIPARSER_BUILD_TESTS=OFF"
+    sources:
+    - type: archive
+      url: https://github.com/uriparser/uriparser/releases/download/uriparser-0.9.4/uriparser-0.9.4.tar.bz2
+      sha256: b7cdabe5611408fc2c3a10f8beecb881a0c7e93ff669c578cd9e3e6d64b8f87b
+    cleanup:
+      - "/include"
+      - "/bin"
+
+  - name: pugixml
+    buildsystem: cmake-ninja
+    build-options:
+      cflags: "-fPIC"
+      cxxflags: "-fPIC"
+    sources:
+    - type: archive
+      url: https://github.com/zeux/pugixml/releases/download/v1.10/pugixml-1.10.tar.gz
+      sha256: 55f399fbb470942410d348584dc953bcaec926415d3462f471ef350f29b5870a
+    cleanup:
+      - "/include"
+      - "*.a"
+
+  - name: poco
+    buildsystem: cmake-ninja
+    sources:
+    - type: archive
+      url: https://github.com/pocoproject/poco/archive/refs/tags/poco-1.11.8-release.tar.gz
+      sha256: a59727335a9bf428dc1289cd8ce84f9c1749c1472a0cd3ae86bec85be23d7cbe
+    cleanup:
+      - "/include"
+      
+  - name: libdaemon
+    buildsystem: autotools
+    sources:
+      - type: git
+        url: https://git.0pointer.net/clone/libdaemon.git
+        commit: 9fcc28e0e8f84968d1fb8b6d544a42efb13803ec
+        disable-shallow-clone: true # dumb http transport does not support shallow capabilities
+    cleanup:
+      - "/include"
+      - "/bin"
+      - "*.a"
+        
+  - name: libevent
+    buildsystem: autotools
+    sources:
+      - type: git
+        url: https://github.com/libevent/libevent
+        tag: release-2.1.12-stable
+    cleanup:
+      - "/include"
+      - "/bin"
+      - "*.a"
+        
+  - name: avahi
+    buildsystem: autotools
+    config-opts:
+      - "--with-distro=none"
+      - "--disable-gtk3"
+      - "--disable-mono"
+      - "--disable-qt5"
+      - "--disable-manpages"
+      - "--disable-python"
+      - "--with-systemdsystemunitdir=no"
+    sources:
+      - type: git
+        url: https://github.com/lathiat/avahi
+        tag: v0.8
+    cleanup:
+      - "/include"
+      - "/bin"
+      - "*.a"
+      
+  
+  - name: libsnappy
+    buildsystem: cmake-ninja
+    config-opts:
+      - "-DBUILD_SHARED_LIBS=ON"
+      - "-DSNAPPY_BUILD_TESTS=OFF"
+      - "-DCMAKE_BUILD_TYPE=Release"
+    sources:
+      - type: git
+        url: https://github.com/google/snappy
+        tag: 1.1.7
+    cleanup:
+      - "/include"
+
+  - name: openframeworks
+    buildsystem: simple
+    build-commands:
+      - mkdir -p ${FLATPAK_DEST}/lib/oF
+      - mv * ${FLATPAK_DEST}/lib/oF
+      - sh ${FLATPAK_DEST}/lib/oF/scripts/linux/compileOF.sh -j$FLATPAK_BUILDER_N_JOBS
+    sources:
+      # the archive includes many libs that are required for compiling and is straighforward, just cloning the git won't work
+      - type: archive
+        url: https://github.com/openframeworks/openFrameworks/releases/download/0.12.0/of_v0.12.0_linux64gcc6_release.tar.gz
```

Comment:
how is this gonna work on aarch64?

Reply:
it won't. I'd like to support x86_64 for this instance and add support for ARM at a latter moment

Reply:
then what does it do when building the aarch64?

Reply:
https://docs.flathub.org/docs/for-app-authors/maintenance/#dont-build-on-aarch64

Reply:
done in the last commit